README.rst

```diff
@@ -19,25 +19,7 @@ For more information about contributing a recipe, read the `documentation`_ on
 the main repository.
 
 Unlike for official recipes, the pull requests for new contrib recipes are
-managed by the community. Pull requests are reviewed by the Symfony Bot and
-automatically merged when the following conditions are met (in that order):
-
-* The pull request title does not contain "WIP";
-
-* The Symfony Bot approved the pull request and reported no validation errors;
-
-* Someone (not the Symfony bot nor the pull request author) approved the pull
-  request;
-
-* The pull request author or the reviewer is a "trusted user". A "trusted user"
-  being one of the following:
-
-  * A Symfony Core Merger;
-
-  * The user is the Github owner for all modified packages;
-
-  * The user is a member of the organizations (publicly visible) for all
-    modified packages.
```

Comment:
Actually, everything is still true except for the last item which should read: "The pull request author or the reviewer is a Symfony Core Merger"

Reply:
This is already the first point ;-) So in fact, the third needs to be deleted - that's what I did now.

Reply:
Why keep a list with one item?

Reply:
*Two* items (GitHub's display here is a little confusing):
  * a Symfony Core Merger;
  * the Github owner for all modified packages.

Reply:
The second one is not true anymore; that's why I proposed "The pull request author or the reviewer is a Symfony Core Merger"

Reply:
Sorry, didn't understand what you meant by "last item" ;-) Fixed now - hopefully :-)